github/Contribution.md

```diff
@@ -0,0 +1,98 @@
+# Contribute to IAI repositories
+
+Everyone is welcome to get involved and we appreciate everybody's contribution.
+Code contributions are not the only way to be involved. 
+Proposing new features or improvement, giving feedback, and improving the documentation are also valuable to improve a library.
+
+## Ways to contribute
+
+There are several ways you can contribute:
+
+  * Fix open issues.
+  * Submit new issues related to desired feature, new ideas, or bugs.
+  * Contribute to the documentation.
+
+## Fix open issues
+
+If you have an idea to fix an open issue, feel free to work on it and open a [Pull Request](#create-a-pull-request).
+
+## Submitting a new issue
+
+Please follow these guidelines when submitting a new bug-related issue or feature request to facilitate the management. 
+
+### Bug-related issue
+
+Before creating a bug-related issue, please make sure it has not been reported before (use the search feature under Issues) to avoid duplicated issues.
```

Comment:
Nit: "to avoid duplicated issues." => "to avoid duplicates."

Reply:
Done.